src/lib/mina/actions/offchain-contract-tests/ExampleContract.ts

```diff
@@ -0,0 +1,89 @@
+import {
```

Comment:
I am okay with leaving it here for now but next time we add larger tests we should add them to the example folder and run them from there - makes it a bit easier to keep the internal code clean yet test it and provide a good example to the community

Reply:
I'd be happy to follow convention, but it doesn't seem like there is one.  This unit-test file was already here and I added onto it.  It seemed like it was getting too big so I broke it out into different files.  I'll follow up with you about how you see o1js being organized and we can work towards that.

Reply:
I'd be happy to follow convention, but it doesn't seem like there is one.  This unit-test file was already here and I added onto it.  It seemed like it was getting too big so I broke it out into different files.  I'll follow up with you about how you see o1js being organized and we can work towards that.

Reply:
I agree, definitely not your fault - just something I noticed with the larger test